qutip/solver/result.py

```diff
@@ -1046,6 +1201,8 @@ def photocurrent(self):
         """
         Average photocurrent or measurement of the evolution.
         """
+        # TODO this is wrong if trajectories have weights
+        # unclear how to implement in case of time-dependent weights
```

Comment:
Does photocurrent make sense for nm_mcsolve?
If it does not, we could raise an error when used with time-dependent weights.

Reply:
It probably does not make sense for nm_mcsolve. I have added an error in the case of time-dependent weights, and removed the tests for it.